### PR TITLE
[quantization] Save `attention_mask` and `position_embeddings`

### PR DESCRIPTION
**Pull request by stamalakhov**
This PR adds `attention_mask` and `position_embeddings` for prefill-decode circle models to inputs to accomodate padding.
According to https://github.com/Samsung/TICO/blob/6ceae5c1ecbead9b7c28fdcc5bdcb8ca24e9e6f7/tico/quantization/wrapq/examples/evaluate_fk_llama_model.py#L234-L236 and https://github.com/Samsung/TICO/blob/6ceae5c1ecbead9b7c28fdcc5bdcb8ca24e9e6f7/tico/quantization/wrapq/examples/evaluate_fk_llama_model.py#L96-L97

 `attention_mask` and `position_embeddings` will be needed to be passed as parameters to prefill-decode models. This PR makes it possible.

<details> <summary> prefill model for Maykeye/TinyLLama-v0</summary>

image

</details>

<details> <summary> decode model for Maykeye/TinyLLama-v0</summary>

image

</details>

models:
[prefill_decode_models_for_Maykeye_TinyLLama_v0.zip](https://github.com/user-attachments/files/27393068/prefill_decode_models_for_Maykeye_TinyLLama_v0.zip)

TICO-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

**Review by mhs4670go — APPROVED** [review]
LGTM
